docs/guides/admin/docs/configuration/transcription.configuration/amberscripttranscripts.md

```diff
@@ -38,155 +38,81 @@ dropdowns below.
-      </configurations>
-    </operation>
-
-    <!-- Merge caption/transcript to existing publication and republish -->
-    <operation id="publish-engage"
-      fail-on-error="true"
-      exception-handler-workflow="partial-error"
-      description="Distribute and publish to engage server">
-      <configurations>
-        <configuration key="download-source-flavors">dublincore/*,security/*</configuration>
-        <configuration key="download-source-tags">engage-download</configuration>
-        <configuration key="strategy">merge</configuration>
-        <configuration key="check-availability">false</configuration>
-      </configurations>
-    </operation>
-
-    <operation
-      id="republish-oaipmh"
-      exception-handler-workflow="partial-error"
-      description="Update recording metadata in default OAI-PMH repository">
-      <configurations>
-        <configuration key="source-flavors">dublincore/*,security/*</configuration>
-        <configuration key="download-source-tags">engage-download</configuration>
-        <configuration key="repository">default</configuration>
-      </configurations>
-    </operation>
-
-    <!-- Archive media package -->
-
-    <operation id="snapshot"
-      fail-on-error="true"
-      exception-handler-workflow="partial-error"
-      description="Archive media package">
-      <configurations>
-        <configuration key="source-flavors">*/*</configuration>
-      </configurations>
-    </operation>
-
-    <!-- Clean up work artifacts -->
-
-    <operation
-        id="cleanup"
-        fail-on-error="false"
-        description="Remove temporary processing artifacts">
-      <configurations>
-        <configuration key="delete-external">true</configuration>
-        <!-- FixMe Don't clean up ACLs until workflow service no longer looks for them in the WFR. -->
-        <configuration key="preserve-flavors">security/*</configuration>
-      </configurations>
-    </operation>
-
-  </operations>
-
-</definition>
-```
+<summary>amberscript-start-transcription.yaml</summary>
+```yaml
+---
+id: amberscript-start-transcription
+title: Start AmberScript Transcription
+tags:
+  - archive
+
+description: Start AmberScript transcription
+
+operations:
+- id: encode
+  fail-on-error: false
+  exception-handler-workflow: partial-error
+  description: Encoding audio for transcription
+  configurations:
+    - source-flavor: "*/source"
+    - target-flavor: audio/mp3
+    - target-tags: transcript
+    - encoding-profile: audio-mp3
+
+- id: amberscript-start-transcription
+  max-attempts: 3
+  retry-strategy: hold
+  fail-on-error: false
+  exception-handler-workflow: partial-error
+  description: Start AmberScript transcription job
+  configurations:
+    - source-tag: transcript
+    - language: de
+    - jobtype: direct
+    - skip-if-flavor-exists: captions/vtt
 
 </details>
 
 <details>
 
-<summary>amberscript-start-transcription.xml</summary>
```

Comment:
This appears to be some kind of merge/squash issue, renders like this:

![image](https://github.com/user-attachments/assets/72acf2cb-c63e-4744-ae6c-8a7f51378ba2)

Reply:
Yep, looks like a merge conflict.
It's fine in the 16.x docs: https://docs.opencast.org/r/16.x/admin/#configuration/transcription.configuration/amberscripttranscripts/

Reply:
Should be fixed now.